filebeat/module/santa/log/manifest.yml

```diff
@@ -7,5 +7,5 @@ var:
   - name: input
```

Comment:
Can you add another log file path above too. Looks like the default has changed to `/var/db/santa/santa.log`.

https://santa.readthedocs.io/en/latest/details/logs/